promitor-agent-scraper/values.yaml

```diff
@@ -13,8 +13,11 @@ image:
   tag:
 
 azureAuthentication:
-  appId: ""
-  appKey: ""
```

Comment:
We'll have to be backward compatible here where the new one is favored over the old one; but we shouldn't break the current experience.